torch/fx/graph_module.py

```diff
@@ -828,7 +839,45 @@ def recompile(self) -> PythonCode:
 
         cls = type(self)
         co_fields = self._graph._co_fields if hasattr(self._graph, "_co_fields") else {}
-        cls.forward = _forward_from_src(self._code, python_code.globals, co_fields)
+
+        if torch.fx.config.run_graph_module_code_from_file:
+
+            debug_dir = torch.fx.config.fx_graph_module_dir
+            if debug_dir is None:
+                debug_dir = tempfile.gettempdir()
+            os.makedirs(debug_dir, exist_ok=True)
+
+            # Generate a unique filename based on module class name and id
+            module_name = f"{FX_GRAPH_MODULE_FILE_PREFIX}{cls.__name__}_{id(self)}"
```

Comment:
don't do this. Instead, hash the source code so you have something like a content addressed store